src/mlpack/tests/main_tests/logistic_regression_test.cpp

```diff
@@ -626,4 +626,25 @@ TEST_CASE_METHOD(LogisticRegressionTestFixture, "LRDecisionBoundaryTest",
 
   // Check that the output changed when the decision boundary moved.
   REQUIRE(arma::accu(output1 != output2) > 0);
+
+  /**
+   * Check that running the binding with print_training_accuracy set to true
+   * does not crash.
+   */
+  TEST_CASE_METHOD(LogisticRegressionTestFixture, "LRPrintTrainingAccuracyTest",
+                  "[LogisticRegressionMainTest][BindingTests]")
+  {
+    constexpr int N = 100;
+    constexpr int D = 5;
+
+    arma::mat trainX = arma::randu<arma::mat>(D, N);
+    arma::Row<size_t> trainY(N, arma::fill::randu);
```

Comment:
```suggestion
    arma::Row<size_t> trainY = arma::randi<arma::Row<size_t>>(N,
        arma::distr_param(0, 1));
```
Technically for type size_t I believe randu will result in a vector of all zeros.  You could instead do this.  (I think the test is fine either way.)